include/ddc/kernels/splines/splines_linear_solver_sparse.hpp

```diff
@@ -114,16 +114,22 @@ unsigned int default_preconditionner_max_block_size() noexcept
 }
 
 /**
- * @brief A sparse Matrix.
+ * @brief A sparse linear problem dedicated to the computation of a spline approximation.
  *
  * The storage format is CSR. Ginkgo is used to perform every matrix and linear solver-related operations.
  *
  * @tparam ExecSpace The Kokkos::ExecutionSpace on which operations related to the matrix are performed.
  */
 template <class ExecSpace>
-class Matrix_Sparse : public Matrix
+class SplinesLinearProblemSparse : public SplinesLinearProblem<ExecSpace>
 {
-    using matrix_sparse_type = gko::matrix::Csr<double, int>;
+public:
+    using typename SplinesLinearProblem<ExecSpace>::MultiRHS;
+    using SplinesLinearProblem<ExecSpace>::size;
+    using SplinesLinearProblem<ExecSpace>::operator<<;
```

Comment:
```suggestion
```

Are these two aliases required to make it work ?

Reply:
`ddc/include/ddc/kernels/splines/splines_linear_solver_sparse.hpp(202): error: identifier "size" is undefined`

Reply:
I see, without this alias we would need to call `this->size()`. Let's keep the alias.